### DISCUSSION TITLE
Dynamic configuration

### DISCUSSION THREAD

**Discussion by vladmelnyk** [quickfixgo/quickfix#492] [Q&A]
Hello.
Depending on holidays we might have days when there is no trading or trading hours is shorter.
But the library (Initiator to be specific) only allows to setup up those values at constructor. 
I was wondering if we can override those settings dynamically?

**Comment by totalys** [comment]
Is hits project forgotten ? I'm not seing recent activities and there are a lot of pull requests pending...

**Comment by grojef** [comment]
with the daily session  configurations, you can invoke the session.stop() method with holidays,otherwise invoke session.start() methd.
